src/app/Layouts/__snapshots__/defaultPageWrapper.test.jsx.snap

```diff
@@ -7,14 +7,11 @@ exports[`defaultPageWrapper should render with children 1`] = `
   >
     <GlobalStyle />
     <RequestContextProvider
-      env="live"
+      bbcOrigin="https://www.bbc.com"
```

Comment:
Why the change to `bbcOrigin`?

Reply:
Its not changing it, but rather passing it to `RequestContextProvider` to determine the `origin` prop that was previously passed in